go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/mattermost/mattermost-plugin-apps
 go 1.16
 
 require (
+	github.com/aws/aws-lambda-go v1.19.1
```

Comment:
My editor didn't liked the nested go modules files and hence I move the two dependencies back into the root one. IMO it makes thing simpler.